packages/twenty-server/src/workspace/workspace-resolver-builder/constants/duplicate-criteria.constants.ts

```diff
@@ -0,0 +1,20 @@
+import { RecordDuplicateCriteria } from 'src/workspace/workspace-query-builder/interfaces/record.interface';
+
+export const duplicateCriteriaCollection: RecordDuplicateCriteria[] = [
+  {
+    objectName: 'company',
+    fieldNames: ['domain'],
+  },
+  {
+    objectName: 'company',
+    fieldNames: ['name'],
+  },
+  {
+    objectName: 'person',
+    fieldNames: ['nameFirstName', 'nameLastName'],
+  },
+  {
+    objectName: 'person',
+    fieldNames: ['email'],
+  },
+];
```

Comment:
BTW: The array contains more criteria than the ticket, is it just for testing ?

Reply:
Yes that was just for the testing purpose :) At the same time i though this might make sense (if i got a duplicate email, then its probably the same person?). Im ok with deleting it though as it was not in the original ticket.

Reply:
https://github.com/twentyhq/twenty/issues/4048#issuecomment-1953135739 Made me think that maybe we can add person.LinkedinLinkUrl as a default too?

Reply:
Good point :) Ive added `person.linkedinLinkUrl` as well.